connector/connector.go

```diff
@@ -128,6 +128,9 @@ type Web3Connector interface {
 	// empty string.
 	InfuraID() string
 
+	// RpcUrl returns the configured eth rpc url to connect to
+	RpcURL() string
```

Comment:
I don't think you want this in this PR, do you?

Reply:
Anyway, if you want to toss it in that's alright. I'll have to remember how to update the charts. It should be `RPCURL`, I think. And we should tighten up the doc string so that it looks like the others: full sentences, proper capitalization, all that.

To be honest, I don't remember why I added the `InfuraID` method, that probably wasn't necessary.

Reply:
Ah, ah, I remember—external things need InfuraID to be able to populate the `web3.html` template. I would just take this method out, actually—no one external really needs to know the RPC URL.